README.md

```diff
@@ -66,7 +66,7 @@ jobs:
     steps:
       - name: Set environment
         id: env-scope
-        uses: im-open/set-environment-variables-by-scope@v1.1.5
+        uses: im-open/set-environment-variables-by-scope@v1.1.6
```

Comment:
Since this is a node20 update, you'll want to increment the minor version.
```suggestion
        uses: im-open/set-environment-variables-by-scope@v1.2.0
```